.github/workflows/master_post_message.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           TEAMS="${{inputs.required-teams}}"
```

Comment:
Why this code is not moved to function?

Reply:
Setting GITHUB_OUTPUT doesn't works if moved to script so kept it as it's since output is the only way for it to be consumed in further steps